rideshare.rb

```diff
@@ -0,0 +1,79 @@
+# set up nested data structure
+rideshare = {
+    DR0001: [
+        {:date => "3rd Feb 2016", :cost => 10, :rider_id => "RD0003", :rating => 3},
+        {:date => "3rd Feb 2016", :cost => 30, :rider_id => "RD0015", :rating => 4},
+        {:date => "5th Feb 2016", :cost => 45, :rider_id => "RD0003", :rating => 2}
+    ],
+    DR0002: [
+        {:date => "3rd Feb 2016", :cost => 25, :rider_id => "RD0073", :rating => 5},
+        {:date => "4th Feb 2016", :cost => 15, :rider_id => "RD0013", :rating => 1},
+        {:date => "5th Feb 2016", :cost => 35, :rider_id => "RD0066", :rating => 3}
+    ],
+    DR0003: [
+        {:date => "4th Feb 2016", :cost => 5, :rider_id => "RD0066", :rating => 5},
+        {:date => "5th Feb 2016", :cost => 50, :rider_id => "RD0003", :rating => 2}
+    ],
+    DR0004: [
+        {:date => "3rd Feb 2016", :cost => 5, :rider_id => "RD0022", :rating => 5},
+        {:date => "4th Feb 2016", :cost => 10, :rider_id => "RD0022", :rating => 4},
+        {:date => "5th Feb 2016", :cost => 20, :rider_id => "RD0073", :rating => 5}
+    ]
+}
+# count each driver's rides
+rideshare.each {|key, value| puts "Driver #{key} went on a total of #{value.count} rides."}
+
```

Comment:
Two comments here:
- I would probably break this out across multiple lines and use `do..end` instead of curly braces
- `key` and `value` are not very descriptive variable names. What about `driver_id` and `rides`?